src/app/components/case/forms/DecisionForm/components/DecisionHeader.tsx

```diff
@@ -1,19 +1,19 @@
-import { useSummonsWithCaseId, useCaseTasks } from "app/state/rest"
+import { useSummonsWithCaseId } from "app/state/rest"
 import useValues from "../hooks/useValues"
 import { DefinitionList } from "@amsterdam/wonen-ui"
 
 type Props = {
-  caseId: Components.Schemas.Case["id"]
+  caseId: Components.Schemas.CaseDetail["id"]
   caseUserTaskId: Components.Schemas.CaseUserTaskWorkdflow["case_user_task_id"]
+  workflows: Components.Schemas.CaseDetail["workflows"]
 }
 
-const DecisionHeader: React.FC<Props> = ({ caseId, caseUserTaskId }) => {
+const DecisionHeader: React.FC<Props> = ({ caseId, caseUserTaskId, workflows }) => {
 
   const [summons, { isBusy: isBusySummons }] = useSummonsWithCaseId(caseId)
```

Comment:
`isBusySummons` can be removed, right?
At row 14 you declare it `isBusy` again... ;)

Reply:
You're right, I'll change it.